packages/components/addon/components/hds/dropdown/toggle.hbs

```diff
@@ -0,0 +1,25 @@
+{{#if (eq @toggle "more")}}
+  <button class={{this.toggleClassNames}} aria-label={{@text}} type="button" ...attributes>
+    <FlightIcon @name="more-horizontal" @size={{if @size "16" "24"}} />
+  </button>
+{{else if (eq @toggle "user")}}
+  <button class={{this.toggleClassNames}} aria-label={{@text}} type="button" ...attributes>
+    <div class="hds-dropdown-toggle__user">
+      {{#if @avatarUrl}}
+        <img src={{@avatarUrl}} alt="" role="presentation" height="32" width="32" />
+      {{else}}
+        <FlightIcon @name="user" @size="24" />
+      {{/if}}
+    </div>
+    <FlightIcon @name="chevron-down" />
+  </button>
+{{else}}
+  <Hds::Button
```

Comment:
[question, non-blocker] When I first spiked out this component, I tried to use the Hds::Button component & you told me not to; can you explain what has changed?